ego_client.py

```diff
@@ -1,57 +1,237 @@
+import json
 class EgoClient(object):
-    def __init__(self, base_url, mocks=None):
-        if mocks is None:
-            mocks = {}
+    daco_policies = {'portal'}
```

Comment:
can we make the daco and cloud policies configurable from a config file / env variables?